src/components/UI/Form/EmojiInput/Editor.module.css

```
@@ -61,16 +60,13 @@
 }
 
 .disabled {
+  composes: Editor;
   position: relative;
-  border: 2px solid rgba(0, 0, 0, 0.23);
-  border-radius: 12px;
   height: 10rem;
   position: relative;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

position is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)